app/models/article.server.model.js

```diff
@@ -25,7 +21,19 @@ var ArticleSchema = new Schema({
 		default: '',
 		trim: true
 	},
-	user: {
+	updatedAt: {
+		type: Date,
+		default: Date.now
+	},
+	updatedBy: {
+		type: Schema.ObjectId,
+		ref: 'User'
+	},
+	createdAt: {
```

Comment:
why `createdAt`/`updatedAt` and not just `created`/`updated`? Those seem pretty clear to me when you also have the `..By` fields.